Modify concepts sections to point at Event-Centric future (#915)
datatrails/datatrails-docs: content/platform/administration/sharing-access-inside-your-tenant/index.md

**Caution:** You will only have access to the `Access Policies` screen if you are an Administrator in your organization.
{{< /caution >}}

Attribute-Based Access Control (ABAC) policies can be used to control access Audit Trails and Events within a single organization.

Specifically, ABAC policies are created by Administrators to share information with Non-Administrators in the same Tenancy.

ABAC policies can be granular, with users only allowed to see single attributes at a time.

> It is possible to control read and write access to Trails based on a combination of properties of the Trail as specified in the Asset attributes, such as type and location (eg: "all models trained in Gondwalaland")

Author: > Still references Asset. Should that be _Event attributes_, or still needs to be Asset 'till we retire Assets all together?

Unfortunately this needs to stay until we have the new system. It's the only way to define sharing policies.

> By default, no Non-Administrators will see any existing Trails or Events unless an Administrator explicitly creates an ABAC policy to allow it.

Author: ```suggestion
By default, new users will not see any existing Trails or Events until an Administrator explicitly creates an ABAC policy to allow them.
```
